mdbenchmark/generate.py

```diff
@@ -245,6 +247,12 @@ def generate(name, cpu, gpu, module, host, min_nodes, max_nodes, time, skip_vali
                     time=time,
                 )
 
+            bundle = mds.discover(directory)
```

Comment:
Don't you have the `bundle` above already? `base_directory` defines it for the given loop -- or was that something different? Also maybe we should stick with `dtr` and try to avoid `mds` usage, if we talk about removing it.